sass/components/form.scss

```diff
@@ -279,27 +285,27 @@
     }
     .dropdown {
         position: absolute;
-        top: 0;
+        top: 35px;
         right: 0;
         opacity: 0;
         pointer-events: none;
-        z-index: +2;
-        width: 100%;
+        z-index: 2;
         max-height: 200px;
         overflow: scroll;
         box-shadow: 0 2px 8px 0 rgba(42, 48, 82, 0.14);
         box-sizing: border-box;
         border-radius: 4px;
         font-size: 14px;
+        width: 272px;
+        transition: opacity 0.2s, top 0.2s;
```

Comment:
Same here, can we use `transform translate` instead?

Reply:
replaced